resources/views/themes/default1/common/setting/system.blade.php

```diff
@@ -458,34 +460,42 @@
           var column = $(this).attr('value');
         
 
-          event.preventDefault();
-          swal({
-              title: `Are you sure to delete this record?`,
-              text: "If you delete this, it will be gone forever.",
-              
-              buttons: true,
-              dangerMode: true,
-          })
-          .then((willDelete) => {
-            if (willDelete) {
+        
+           if (confirm("Are you sure you want to suspend this user?"))
```

Comment:
Also where ever there are messages like this, when you work on it can you move it to the lang file